index.html

```diff
@@ -62,5 +62,184 @@ <h1>Hi👋, I'm Cephas Tokoli</h1>
          </div>     
         </div>   
     </section>
+    <section class="grid">
+            <div>
+                <h2 class="headings">Projects</h2>
+            </div>
+     
+            <div class="project">
+                <div class="laptop-img">
+                    <img src="images/laptop.png" alt="laptop">
+                </div>
+                <div class="project-details">
+                    <div class="tech-details">
+                        <p class="project-name">Project name goes here</p>
+                    <div class="tech-stack">
+                        <p>HTML</p>
+                        <p>Ruby on Rails</p>
+                        <p>Javascript</p>
+                    </div>
```

Comment:
- [ ] Please make sure that the list of project technologies are a list `ul` and each of them a `li` as given in the project requirements 📌

  > _Please use `ul` and `li` for categories in "Work" section. Html, CSS, and JavaScript should be list items._

- [ ] Also, I think it would be better to reduce padding from top and bottom a little bit to match the given Figma design. See the images below 👇

Your design | Figma design 
---| ---|
![image](https://user-images.githubusercontent.com/106119105/236015250-522327a1-b0da-475e-aabd-abc27e608f1e.png) | ![image](https://user-images.githubusercontent.com/106119105/236015182-c7f99c46-0fd0-4d4a-960c-2b5f5b58f0de.png)